src/frontend/utils/theme/theme.ts

```diff
@@ -15,6 +15,7 @@ const colorsGeneric = {
   'blue-focus': '#0249a4',
```

Comment:
in the commit message, small feedback that I had starting working here: No need to start a new line at each sentence, but actually try to use all the length possible
Define three different fonts to be used in the chat. The styles used are based 
on the mockup.